src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+const Big = require('big-js');
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    try {
+      return one.mod(two).toString();
+    } catch (err) {
+      return "Can't find modulo as can't divide by 0.";
+    }
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```

Comment:
- [ ] [OPTIONAL] Great implementation. However, for a better UI, let's ensure that the app is aligned to the center of the page.
### Your Design
![Screenshot from 2022-05-25 18-16-21](https://user-images.githubusercontent.com/58692590/170324498-1e32bab0-8102-425d-b9e4-e13250445ff1.png)

Reply:
Please check your browser, or try it in another browser because it looks just fine.
![margin](https://user-images.githubusercontent.com/34779928/170327005-8e42900f-8b19-4fcb-890d-2a68638aeef8.gif)